### ISSUE TITLE
Goldmark attribute panic

### ISSUE THREAD

**Issue by bep** [gohugoio/hugo#9819]
This gets a nilpointer:

```bash
-- config.toml --
[markup]
[markup.goldmark]
[markup.goldmark.parser]
autoHeadingID = true
autoHeadingIDType = "github"
[markup.goldmark.parser.attribute]
block = true
title = true
-- content/p1.md --
---
title: "p1"
---
asdf
: {#myid}

-- layouts/_default/single.html --
{{ .Content }}
```

**Comment by bowman2001** [comment]
Another problem, I haven’t filed an issue for, may be related. An attribute without any preceding Markdown content always leads to an error. This kind of false syntax may accidentally happen like the missing content in the definition list item of your example.

**Comment by bep** [comment]
Possible, but this issue was fixed in d7b54a4c37c39fce60c25a745bae1d5987b2b966
